app/models/course.rb

```diff
@@ -172,6 +174,45 @@ def allocations_report_fields
     ]
   end
 
+  def spreadsheet_columns
+    [
+      ['Academic Year', '2020/21'],
+      ['Requested By (Name)', accrediting_provider&.provider_name],
+      ['Requested by (UKPRN)', ''],
+      ['Partner ITT Provider (Name)', provider.provider_name],
+      ['Partner ITT provider (UKPRN)', provider.organisations.first&.school_nctl_organisation&.urn],
+      ['Allocation Subject', allocation_subjects.join(' | ')],
+      ['Route', program_type],
+      ['Course Aim', qualification],
```

Comment:
Please double check these for sanity, they are massaged from the `allocations_report_fields` above